### DISCUSSION TITLE
"punctuate" parameter for the Whisper Cloud model

### DISCUSSION THREAD

**Discussion by eilonmore** [deepgram/community#757] [General help] [answered]
Hey there!

Does anyone know if there is a way to punctuate a pre-recorded transcript with the Whisper cloud model? I tried to pass "punctuate" as a parameter, just like I do with the Nova model. While it works with the Nova model, it doesn't seem to work with the Whisper cloud model.

Any ideas?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @eilonmore, you can use our [Smart Formatting](https://developers.deepgram.com/docs/smart-format) feature, which is also available on our managed Whisper model. The parameter is `smart_format=true`. Smart Formatting includes punctuation, as well as other formatting to make transcripts more "human-readable" and tidy.

**Reply by eilonmore** [reply]
Thank you @jkroll-deepgram! I will give it a try.
